Moblin/Platforms/Kick/KickPusher.swift

```diff
@@ -290,11 +357,27 @@ final class KickPusher: NSObject {
 
     private func handleChatMessageEvent(data: String) throws {
         let event = try decodeChatMessageEvent(data: data)
+        var badgeUrls: [URL] = []
+        for badge in event.sender.identity.badges {
+            if badge.type == KickBadges.BadgeType.subscriber, let months = badge.count {
+                if let badgeUrlString = badges.getSubscriberBadgeUrl(months: months) {
+                    if let badgeUrl = URL(string: badgeUrlString) {
+                        badgeUrls.append(badgeUrl)
+                    }
+                }
+            } else if let badgeUrlString = badges.getBadgeUrl(for: badge.type) {
+                if let badgeUrl = URL(string: badgeUrlString) {
```

Comment:
I prefer to save CPU by only converting the URL strings ti the URL type once, not every time the badge is used.